vmm/src/vm.rs

```diff
@@ -743,6 +743,8 @@ impl Vm {
                 .unwrap()
                 .balloon_resize(desired_ram_w_balloon)
                 .map_err(Error::MemoryManager)?;
+            let balloon_size = self.config.lock().unwrap().memory.size - desired_ram_w_balloon;
+            self.config.lock().unwrap().memory.balloon_size = balloon_size;
```

Comment:
Please add a comment on top of this, explaining that we update the configuration value for the balloon size to ensure a reboot would use the right value.

Reply:
Added.